src/uu/more/src/more.rs

```diff
@@ -105,25 +105,31 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
             let file = Path::new(file);
             if file.is_dir() {
                 terminal::disable_raw_mode().unwrap();
-                return Err(UUsageError::new(
-                    1,
+                show!(UUsageError::new(
+                    0,
                     format!("{} is a directory.", file.quote()),
                 ));
+                terminal::enable_raw_mode().unwrap();
+                continue;
             }
             if !file.exists() {
                 terminal::disable_raw_mode().unwrap();
-                return Err(USimpleError::new(
-                    1,
+                show!(USimpleError::new(
+                    0,
                     format!("cannot open {}: No such file or directory", file.quote()),
                 ));
+                terminal::enable_raw_mode().unwrap();
+                continue;
             }
             let opened_file = match File::open(file) {
                 Err(why) => {
                     terminal::disable_raw_mode().unwrap();
```

Comment:
I don't actually know why it stays in raw mode, but it seems unrelated to this change, so let's merge it!